src/services/server.ts

```diff
@@ -15,6 +16,7 @@ const ROUTES: FastifyPluginCallback<RouteOptions>[] = [
   orgRoutes,
   memberRoutes,
   memberPublicKey,
+  memberKeyImportToken,
```

Comment:
Heads-up: Git won't report a merge conflict, but if #89 is merged first, things will break unless we manually move this to `member.routes.ts`.